src/Core/NetPad.Domain/Scripts/Script.cs

```diff
@@ -81,6 +81,11 @@ public bool IsDirty
 
     public bool IsNew => Path == null;
 
+    public void SetConfig(ScriptConfig config)
```

Comment:
Change this method to `UpdateConfig` and the method should "copy" `ScriptConfig` values by setting each property individually. The current code shares the same `ScriptConfig` reference with the script it was taken from. One change there, will reflect on this script too.

Also, since we will be updating the `ScriptConfig` not resetting its value, let's also remove the private setter on the property.